[Bug] Broken Sendstring LUT Headers: Slovak and Lithuanian QWERTY

### Describe the Bug

The two Sendstring LUT headers `sendstring_lithuanian_qwerty.h` and `sendstring_slovak.h` causes builds to fail. All other Sendstring LUT headers work just fine!

I would fix it if I understood what's going wrong. So, in lieu of a nice fix, I can only report the problem here.

**Slovak sendstring build errors:**
```
Compiling: quantum/keymap_introspection.c                                                          In file included from quantum/quantum.h:40,
                 from ./.build/obj_summacogni_mathpad_mp1a_default/src/default_keyboard.h:28,
                 from ./keyboards/summacogni/mathpad/mp1a/keymaps/default/keymap.c:15,
                 from quantum/keymap_introspection.c:9:
quantum/quantum_keycodes.h:46:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '544' to '32' [-Werror=overflow]
   46 | #define LSFT(kc) (QK_LSFT | (kc))
      |                  ^
quantum/quantum_keycodes.h:55:15: note: in expansion of macro 'LSFT'
   55 | #define S(kc) LSFT(kc)
      |               ^~~~
quantum/keymap_extras/keymap_slovak.h:88:17: note: in expansion of macro 'S'
   88 | #define SK_3    S(SK_SCAR) // 3
      |                 ^
quantum/keymap_extras/sendstring_slovak.h:111:59: note: in expansion of macro 'SK_3'
  111 |     SK_X,    SK_Y,    SK_Z,    SK_F,    SK_Q,    SK_G,    SK_3,    SK_MINS,
      |                                                           ^~~~
quantum/quantum_keycodes.h:46:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '548' to '36' [-Werror=overflow]
   46 | #define LSFT(kc) (QK_LSFT | (kc))
      |                  ^
quantum/quantum_keycodes.h:55:15: note: in expansion of macro 'LSFT'
   55 | #define S(kc) LSFT(kc)
      |               ^~~~
quantum/keymap_extras/keymap_slovak.h:92:17: note: in expansion of macro 'S'
   92 | #define SK_7    S(SK_YACU) // 7
      |                 ^
quantum/keymap_extras/sendstring_slovak.h:113:5: note: in expansion of macro 'SK_7'
  113 |     SK_7,    SK_A,    SK_B,    SK_C,    SK_D,    SK_E,    SK_F,    SK_G,
      |     ^~~~
quantum/quantum_keycodes.h:46:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '542' to '30' [-Werror=overflow]
   46 | #define LSFT(kc) (QK_LSFT | (kc))
      |                  ^
quantum/quantum_keycodes.h:55:15: note: in expansion of macro 'LSFT'
   55 | #define S(kc) LSFT(kc)
      |               ^~~~
quantum/keymap_extras/keymap_slovak.h:86:17: note: in expansion of macro 'S'
   86 | #define SK_1    S(SK_PLUS) // 1
      |                 ^
quantum/keymap_extras/sendstring_slovak.h:119:59: note: in expansion of macro 'SK_1'
  119 |     SK_X,    SK_Y,    SK_Z,    SK_B,    SK_W,    SK_N,    SK_1,    KC_DEL
      |                                                           ^~~~
cc1.exe: all warnings being treated as errors
 [ERRORS]
```


**Lithuanian QWERTY sendstring build errors:**
```
Compiling: quantum/keymap_introspection.c                                                          In file included from quantum/quantum.h:40,
                 from ./.build/obj_summacogni_mathpad_mp1a_default/src/default_keyboard.h:28,
                 from ./keyboards/summacogni/mathpad/mp1a/keymaps/default/keymap.c:15,
                 from quantum/keymap_introspection.c:9:
quantum/quantum_keycodes.h:71:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '5150' to '30' [-Werror=overflow]
   71 | #define RALT(kc) (QK_RALT | (kc))
      |                  ^
quantum/quantum_keycodes.h:74:18: note: in expansion of macro 'RALT'
   74 | #define ALGR(kc) RALT(kc)
      |                  ^~~~
quantum/keymap_extras/keymap_lithuanian_qwerty.h:96:17: note: in expansion of macro 'ALGR'
   96 | #define LT_1    ALGR(LT_AOGO) // 1
      |                 ^~~~
quantum/keymap_extras/sendstring_lithuanian_qwerty.h:57:14: note: in expansion of macro 'LT_1'
   57 |     KC_SPC,  LT_1,    LT_QUOT, LT_3,    LT_4,    LT_5,    LT_7,    LT_QUOT,
      |              ^~~~
quantum/quantum_keycodes.h:71:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '5152' to '32' [-Werror=overflow]
   71 | #define RALT(kc) (QK_RALT | (kc))
      |                  ^
quantum/quantum_keycodes.h:74:18: note: in expansion of macro 'RALT'
   74 | #define ALGR(kc) RALT(kc)
      |                  ^~~~
quantum/keymap_extras/keymap_lithuanian_qwerty.h:98:17: note: in expansion of macro 'ALGR'
   98 | #define LT_3    ALGR(LT_EOGO) // 3
      |                 ^~~~
quantum/keymap_extras/sendstring_lithuanian_qwerty.h:57:32: note: in expansion of macro 'LT_3'
   57 |     KC_SPC,  LT_1,    LT_QUOT, LT_3,    LT_4,    LT_5,    LT_7,    LT_QUOT,
      |                                ^~~~
quantum/quantum_keycodes.h:71:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '5153' to '33' [-Werror=overflow]
   71 | #define RALT(kc) (QK_RALT | (kc))
      |                  ^
quantum/quantum_keycodes.h:74:18: note: in expansion of macro 'RALT'
   74 | #define ALGR(kc) RALT(kc)
      |                  ^~~~
quantum/keymap_extras/keymap_lithuanian_qwerty.h:99:17: note: in expansion of macro 'ALGR'
   99 | #define LT_4    ALGR(LT_EDOT) // 4
      |                 ^~~~
quantum/keymap_extras/sendstring_lithuanian_qwerty.h:57:41: note: in expansion of macro 'LT_4'
   57 |     KC_SPC,  LT_1,    LT_QUOT, LT_3,    LT_4,    LT_5,    LT_7,    LT_QUOT,
      |                                         ^~~~
quantum/quantum_keycodes.h:71:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '5154' to '34' [-Werror=overflow]
   71 | #define RALT(kc) (QK_RALT | (kc))
      |                  ^
quantum/quantum_keycodes.h:74:18: note: in expansion of macro 'RALT'
   74 | #define ALGR(kc) RALT(kc)
      |                  ^~~~
quantum/keymap_extras/keymap_lithuanian_qwerty.h:100:17: note: in expansion of macro 'ALGR'
  100 | #define LT_5    ALGR(LT_IOGO) // 5
      |                 ^~~~
quantum/keymap_extras/sendstring_lithuanian_qwerty.h:57:50: note: in expansion of macro 'LT_5'
   57 |     KC_SPC,  LT_1,    LT_QUOT, LT_3,    LT_4,    LT_5,    LT_7,    LT_QUOT,
      |                                                  ^~~~
quantum/quantum_keycodes.h:71:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '5156' to '36' [-Werror=overflow]
   71 | #define RALT(kc) (QK_RALT | (kc))
      |                  ^
quantum/quantum_keycodes.h:74:18: note: in expansion of macro 'RALT'
   74 | #define ALGR(kc) RALT(kc)
      |                  ^~~~
quantum/keymap_extras/keymap_lithuanian_qwerty.h:102:17: note: in expansion of macro 'ALGR'
  102 | #define LT_7    ALGR(LT_UOGO) // 7
      |                 ^~~~
quantum/keymap_extras/sendstring_lithuanian_qwerty.h:57:59: note: in expansion of macro 'LT_7'
   57 |     KC_SPC,  LT_1,    LT_QUOT, LT_3,    LT_4,    LT_5,    LT_7,    LT_QUOT,
      |                                                           ^~~~
quantum/quantum_keycodes.h:71:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '5157' to '37' [-Werror=overflow]
   71 | #define RALT(kc) (QK_RALT | (kc))
      |                  ^
quantum/quantum_keycodes.h:74:18: note: in expansion of macro 'RALT'
   74 | #define ALGR(kc) RALT(kc)
      |                  ^~~~
quantum/keymap_extras/keymap_lithuanian_qwerty.h:103:17: note: in expansion of macro 'ALGR'
  103 | #define LT_8    ALGR(LT_UMAC) // 8
      |                 ^~~~
quantum/keymap_extras/sendstring_lithuanian_qwerty.h:59:23: note: in expansion of macro 'LT_8'
   59 |     LT_9,    LT_0,    LT_8,    LT_ZCAR, LT_COMM, LT_MINS, LT_DOT,  LT_SLSH,
      |                       ^~~~
quantum/quantum_keycodes.h:46:18: error: unsigned conversion from 'int' to 'unsigned char' changes value from '5678' to '46' [-Werror=overflow]
   46 | #define LSFT(kc) (QK_LSFT | (kc))
      |                  ^
quantum/quantum_keycodes.h:55:15: note: in expansion of macro 'LSFT'
   55 | #define S(kc) LSFT(kc)
      |               ^~~~
quantum/keymap_extras/keymap_lithuanian_qwerty.h:114:17: note: in expansion of macro 'S'
  114 | #define LT_PLUS S(ALGR(LT_ZCAR)) // +
      |                 ^
quantum/keymap_extras/sendstring_lithuanian_qwerty.h:63:50: note: in expansion of macro 'LT_PLUS'
   63 |     LT_UMAC, LT_9,    LT_SCLN, LT_SCLN, LT_COMM, LT_PLUS, LT_DOT,  LT_SLSH,
      |                                                  ^~~~~~~
cc1.exe: all warnings being treated as errors
 [ERRORS]
```

### Keyboard Used

Custom device (Summa Cogni Mathpad)

### Link to product page (if applicable)

_No response_

### Operating System

Windows

### qmk doctor Output

```text

```

### Is AutoHotKey / Karabiner installed

- [ ] AutoHotKey (Windows)
- [ ] Karabiner (macOS)

### Other keyboard-related software installed

_No response_

### Additional Context

_No response_

--- a/quantum/keymap_extras/sendstring_lithuanian_qwerty.h
+++ b/quantum/keymap_extras/sendstring_lithuanian_qwerty.h
@@ -54,13 +54,13 @@ const uint8_t ascii_to_keycode_lut[128] PROGMEM = {
     XXXXXXX, XXXXXXX, XXXXXXX, KC_ESC,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
 
     //       !        "        #        $        %        &        '
-    KC_SPC,  LT_1,    LT_QUOT, LT_3,    LT_4,    LT_5,    LT_7,    LT_QUOT,
+    KC_SPC,  LT_AOGO, LT_QUOT, LT_EOGO, LT_EDOT, LT_IOGO, LT_UOGO, LT_QUOT,
     // (     )        *        +        ,        -        .        /
-    LT_9,    LT_0,    LT_8,    LT_ZCAR, LT_COMM, LT_MINS, LT_DOT,  LT_SLSH,
+    LT_9,    LT_0,    LT_UMAC, LT_ZCAR, LT_COMM, LT_MINS, LT_DOT,  LT_SLSH,
     // 0     1        2        3        4        5        6        7
     LT_0,    LT_AOGO, LT_CCAR, LT_EOGO, LT_EDOT, LT_IOGO, LT_SCAR, LT_UOGO,
     // 8     9        :        ;        <        =        >        ?
-    LT_UMAC, LT_9,    LT_SCLN, LT_SCLN, LT_COMM, LT_PLUS, LT_DOT,  LT_SLSH,
+    LT_UMAC, LT_9,    LT_SCLN, LT_SCLN, LT_COMM, LT_ZCAR, LT_DOT,  LT_SLSH,
     // @     A        B        C        D        E        F        G
     LT_CCAR, LT_A,    LT_B,    LT_C,    LT_D,    LT_E,    LT_F,    LT_G,
     // H     I        J        K        L        M        N        O

--- a/quantum/keymap_extras/sendstring_slovak.h
+++ b/quantum/keymap_extras/sendstring_slovak.h
@@ -108,13 +108,13 @@ const uint8_t ascii_to_keycode_lut[128] PROGMEM = {
     // P     Q        R        S        T        U        V        W
     SK_P,    SK_Q,    SK_R,    SK_S,    SK_T,    SK_U,    SK_V,    SK_W,
     // X     Y        Z        [        \        ]        ^        _
-    SK_X,    SK_Y,    SK_Z,    SK_F,    SK_Q,    SK_G,    SK_3,    SK_MINS,
+    SK_X,    SK_Y,    SK_Z,    SK_F,    SK_Q,    SK_G,    SK_SCAR, SK_MINS,
     // `     a        b        c        d        e        f        g
-    SK_7,    SK_A,    SK_B,    SK_C,    SK_D,    SK_E,    SK_F,    SK_G,
+    SK_YACU, SK_A,    SK_B,    SK_C,    SK_D,    SK_E,    SK_F,    SK_G,
     // h     i        j        k        l        m        n        o
     SK_H,    SK_I,    SK_J,    SK_K,    SK_L,    SK_M,    SK_N,    SK_O,
     // p     q        r        s        t        u        v        w
     SK_P,    SK_Q,    SK_R,    SK_S,    SK_T,    SK_U,    SK_V,    SK_W,
     // x     y        z        {        |        }        ~        DEL
-    SK_X,    SK_Y,    SK_Z,    SK_B,    SK_W,    SK_N,    SK_1,    KC_DEL
+    SK_X,    SK_Y,    SK_Z,    SK_B,    SK_W,    SK_N,    SK_PLUS, KC_DEL
 };

